src/lib/venueTimezone.ts

```
@@ -0,0 +1,116 @@
+      best = { tz: c.tz, dist };
+    }
+  }
+  return best?.tz ?? null;
+}
+
+export function resolveVenueTimezone(event: TimezoneAwareEvent): string {
+  if (event.venue_timezone) return event.venue_timezone;
+  const venues = event.venues;
+  const joinedTz = Array.isArray(venues)
+    ? venues[0]?.timezone
+    : venues?.timezone;
```

ESLint

[error] Replace `⏎····?·venues[0]?.timezone⏎···` with `·?·venues[0]?.timezone`

(prettier/prettier)